kubemarine/procedures/backup.py

```diff
@@ -727,7 +727,7 @@ def create_context(cli_arguments: List[str] = None) -> dict:
 
     '''
 
-    parser = flow.new_procedure_parser(cli_help, tasks=tasks)
+    parser = flow.new_procedure_parser(cli_help, tasks=tasks, optional_config=True)
```

Comment:
Let's notice in documentation, if procedure file is optional

Reply:
Added information about the procedure file and emphasized when it is required or optional. Also made some other fixes of doc
https://github.com/Netcracker/KubeMarine/pull/526/commits/e853c9a51c750b72f5e0aa3563e5bebab3c97822